kong/runloop/balancer.lua

```diff
@@ -108,7 +103,7 @@ do
   _load_upstream_into_memory = load_upstream_into_memory
 
   get_upstream_by_id = function(upstream_id)
-    local upstream_cache_key = "balancer:upstreams:" .. upstream_id
+    local upstream_cache_key = "upstreams:" .. upstream_id
```

Comment:
@bungle why did this line in the balancer used a different key? `"balancer:upstreams:" .. upstream_id`

this change here allows upstreams to be warmed up